WordPress/Sniffs/WP/I18nSniff.php

```diff
@@ -185,21 +202,40 @@ public function process_token( $stack_ptr ) {
 			}
 		}
 
-		if ( '_' === $token['content'] ) {
+		// Prevent exclusion of the i18n group.
+		$this->exclude = '';
+
+		// Note: this will give false positives if a method called `_` is used.
+		if ( '_' === $this->tokens[ $stack_ptr ]['content'] ) {
```

Comment:
Is there a particular reason not to move this check as well, other than the fact that `_` isn't a part of the `i18n` group? Couldn't it be added as part of a second group and checked in the `process_matched_token()` method? That way it wouldn't catch false-positives from `_()` methods either. (Which admittedly seems unlikely, but I see no reason not to exclude that possibility.)

Reply:
> Is there a particular reason not to move this check as well, other than the fact that _ isn't a part of the i18n group?

I didn't move for exactly that reason :grin: 

> Couldn't it be added as part of a second group and checked in the process_matched_token() method? That way it wouldn't catch false-positives from _() methods either.

I think there are more elegant ways to do this, I'll have a fiddle with it.

Reply:
Reading your comment again, I now understand what you mean by "second group". Yes, that would actually work perfectly. Changing things now.

Reply:
Done. Added second commit. Feel free to squash on merge.

I've also moved the other check which checks for a particular function down to below the parenthesis check as an extra safeguard against false positives.